### ISSUE TITLE
Confusing "invitation" language between "Share Access" button and admin panel

### ISSUE THREAD

**Issue by pdehaye** [sandstorm-io/sandstorm#1207]
In the admin console there are three types of users: "admin", "guest" and "invited user". 

On the other hand, the "Share Access" button offers two options: 
 - "Get a shareable link" (URL)
 - "Invite user" (by email)

The language of the second is problematic, according to me: if I invite a user through the second option, they might become Guest, not Invited User. 

I suggest: 
 - In the Admin panel: "Admin", "Resident User" or "Guest" (think access control at the entrance of a residential condo)
 - In the "Share Access" panel: "Invite user" -> "Email access key"

**Comment by paulproteus** [comment]
I agree with your general concern! I'll see if I have any other specific
suggestions on the wording shortly, after the weekend, and would be happy
to submit patches.

I imagine @phildini also may have opinions on this.

**Comment by paulproteus** [comment]
This issue may seem small, but note that if we change some terminology, we should make sure to update:

- Comments in the source code, and

- Docs in https://docs.sandstorm.io/en/latest/ , and

- The web UI that users see (which is what @pdehaye initially mentioned) and

- The Sandstorm website.

**Comment by pdehaye** [comment]
Of course! But you know how to goes... The more you wait...

**Comment by paulproteus** [comment]
Ya (-: this is not an objection, merely a note-to-self on how to do a good job fixing this!
